### PR TITLE
NoHeader Test

### PR DESCRIPTION
**Pull request by futobarna**
Created a test for --no-header option and template files with _ROW_ and _UUID_ variables.

### CODE REVIEW

**Comment by martinhuszti** [comment]
Az LGTM miért nem fut le?

File: examples/noheadertest/cars.csv [hunk 1]
```diff
@@ -0,0 +1,5 @@
+1997,Ford,E350,"ac, abs, moon",3000.00
+1999,Chevy,"Venture ""Extended Edition""","",4900.00
+1999,Chevy,"Venture ""Extended Edition, Very Large""",,5000.00
+1996,Jeep,Grand Cherokee,"MUST SELL!
+air, moon roof, loaded",4799.00
```
Comment (martinhuszti) [inline examples/noheadertest/cars.csv]: Cars are boring now, but OK. :)

File: examples/noheadertest/template.ttl [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+:Offer-#{_UUID_} a gr:Offering ;
+    rdfs:comment "${3}" ;
+    gr:includes :Car-${_ROW_} ;
+    gr:hasBusinessFunction gr:Sell ;
+    gr:hasPriceSpecification _:price .
+    
+_:price a gr:UnitPriceSpecification ;
+    gr:hasCurrency "USD"^^xsd:string ;
+    gr:hasCurrencyValue "${4}"^^xsd:float .
```
Comment (martinhuszti) [inline examples/noheadertest/template.ttl]: Prefect template

File: src/hu/bme/mit/budgetanomalia/test/TestNoHeader.java [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    private void init(){
+        Cli.<Runnable> builder("csv2rdf").withDescription("Converts a CSV file to RDF based on a given template")
+                .withDefaultCommand(CSV2RDF.class).withCommand(CSV2RDF.class).withCommand(Help.class)
+                .build().parse(templateFile, csvFile, outputFile, "--no-header").run();
+    }
+
+    private void init2(){
+        Cli.<Runnable> builder("csv2rdf").withDescription("Converts a CSV file to RDF based on a given template")
+                .withDefaultCommand(CSV2RDF.class).withCommand(CSV2RDF.class).withCommand(Help.class)
+                .build().parse(templateFile2, csvFile, outputFile2, "--no-header").run();
```
Comment (martinhuszti) [inline src/hu/bme/mit/budgetanomalia/test/TestNoHeader.java]: Init1 and 2 can be merged, with an input parameter specificing the output file.

File: src/hu/bme/mit/budgetanomalia/test/TestNoHeader.java [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+    @Test
+    public void testFileIsCreated2() {
+        init2();
+        File f = new File(outputFile2);
+        assertNotNull(f);
+    }
+
+    @Test
+    public void testFileNameIsCorrect2() {
```
Comment (martinhuszti) [inline src/hu/bme/mit/budgetanomalia/test/TestNoHeader.java]: I miss some comments to the tests.

**Comment by suledani97** [comment]
Looks good to me.

**Pull request merged**